### PR TITLE
Update `color`, restore `PartialEq` for `Brush`, `BrushRef`

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

File: src/brush.rs [hunk 1]
```diff
@@ -8,7 +8,7 @@ use color::{AlphaColor, ColorSpace, DynamicColor, OpaqueColor, Srgb};
 /// Describes the color content of a filled or stroked shape.
 ///
 /// See also [`BrushRef`] which can be used to avoid allocations.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq)]
```
Comment (DJMcNab) [inline src/brush.rs:11]: Maybe? we should caveat this `PartialEq` in the docs

Comment (waywardmonkeys) [reply src/brush.rs:11]: Without this, you can't use `peniko::Brush` with Parley and this is just restoring it back to how it was before all of the color changes.

What caveat do you want in there?

Comment (waywardmonkeys) [reply src/brush.rs:11]: Going to merge this for now. If we do a write up / coverage of equality in a coherent fashion this would get picked up then.

Comment (DJMcNab) [reply src/brush.rs:11]: Mostly that it doesn't do a perceptual equivalence check. The mistake I made here was thinking that the Color is a dynamic colour, but actually it is in a fixed colour space.

Comment (waywardmonkeys) [reply src/brush.rs:11]: There's also that `f32` in `Image` and all of the various values in the color stops for gradients. Equality can have a discussion, but it needs to be in the right place and thought out.

**Pull request merged**